stdlib/LinearAlgebra/test/symmetric.jl

```diff
@@ -592,4 +596,11 @@ end
     @test Symmetric(B) == Symmetric(Matrix(B))
 end
 
+@testset "symmetric()/hermitian() for Numbers" begin
+    @test symmetric(1, :U) == 1
+    @test symmetric_type(Int) == Int
+    @test hermitian(1, :U) == 1
+    @test hermitian_type(Int) == Int
```

Comment:
These four commands are not exported, and require specialization with `LinearAlgebra.symmetric` etc.

Reply:
Derp, missed that somehow. Thanks!